### PR TITLE
Write a basic guide for AI TSing using AI and PS

### PR DESCRIPTION
**Pull request by gsk143**
I have written a basic guide for masking using AI and PS. Please let me know if this helps.
I am not a programmer and my English isn't great so please excuse any mistakes.

### CODE REVIEW

File: src/typesetting/PSandAI.md
Comment (cid-chan) [inline src/typesetting/PSandAI.md]: `smol`

File: src/typesetting/PSandAI.md
Comment (cid-chan) [inline src/typesetting/PSandAI.md]: A space is missing between PS and AI.

File: src/typesetting/PSandAI.md
Comment (cid-chan) [inline src/typesetting/PSandAI.md]: `all the software`

**Review by cid-chan — DISMISSED** [review]

**Comment by gsk143** [comment]
@cid-chan @OrangeChannel Any chance that this gets merged? I have seen this go around on fansubbing servers and would be nice to have it site and I also plan to write timing guide next if this gets merged :) Thanks

**Comment by OrangeChannel** [comment]
I will take a look soon. Not sure who else is active here but I would want to double check with anyone experienced with the content before I merge.

**Comment by FichteFoll** [comment]
Dave and myself were (?) the de-facto maintainers of this project. I wish I was more active, but I keep having other more important things on my plate, so I apologize for the lack of review activity on my part. I don't to want to make promises I can't keep, but I can say that I have this project in the back of my head.

I've never used AI myself either, but I generally give PR authors the benefit of the doubt.

File: src/typesetting/PSandAI.md
Comment (FichteFoll) [inline src/typesetting/PSandAI.md]: Using "I" in a guide that is maintained by a community is usually a bad idea, unless the author of a page is clearly denoted, because the author may change across pages and sometimes even within the same page. 

Please either add something that allows the reader to identify the author as a single entity and allows them to make the connection to someone in particular (read: you), or rephrase everything to be more general, e.g. by using passive forms like "Before starting, a small warning is warranted." or use "we" to refer to the contributors collectively. Choose whatever you prefer since I don't have a strong opinion on it yet, but know that others may contribute to this page as they and the reviewer see fit.

File: src/typesetting/PSandAI.md
Comment (FichteFoll) [inline src/typesetting/PSandAI.md]: You should never have two headings of the same level next to each other. This should probably be a h2 or be combined into one heading.

File: src/typesetting/imageTrace.md
Comment (FichteFoll) [inline src/typesetting/imageTrace.md]: Because the enumeration of the images is evidently not in order, please use more descriptive names for the files, which will also be more future-proof.

File: src/typesetting/maskingProcedure.md
Comment (FichteFoll) [inline src/typesetting/maskingProcedure.md]: Breaking a line between "will be" and "doing" makes no sense. There are a couple other instances with questionable line breaks, but this stood out the most to me.

Please try to rebreak the lines at more structural berak points within a sentence, such as between subject, predicate, object, or around interjections and similar. Kind of how you would break long code lines between parameters, chained function calls, or around binary operators.

**Review by FichteFoll — CHANGES_REQUESTED** [review]
I took a brief look at some parts because at this point I might as well and left a few comments. They are primarily concerning the form of the text, not the contents themselves, which I didn't check at all, but should be a good start for refining it.

**Comment by OrangeChannel** [comment]
The links and sidebar all look good, all images and formatting seems to be good except for the header level that Fichte already mentioned. Once those comments are addressed I can go through and just do a quick grammar/spelling check.

**Comment by gsk143** [comment]
@OrangeChannel  Thanks for the offer for English check since I am not good at it. I will address the other issues tomorrow once I get back to home.

File: src/typesetting/PSandAI.md
Comment (gsk143) [reply src/typesetting/PSandAI.md]: I have mentioned myself at the beginning and made clear it is me and I would like to keep it that way through out my guides (Mostly I will write TS/Timing stuff in future)

File: src/typesetting/PSandAI.md
Comment (gsk143) [reply src/typesetting/PSandAI.md]: I have combined it since it makes more sense to me that way

File: src/typesetting/imageTrace.md
Comment (gsk143) [reply src/typesetting/imageTrace.md]: Made the file name more clear now and used respective directories for ease

File: src/typesetting/maskingProcedure.md
Comment (gsk143) [reply src/typesetting/maskingProcedure.md]: This is something I cannot do since I am not from English background and not good enough in it to do it myself. If someone do it I would appreciate it

**Comment by gsk143** [comment]
Sorry for being late, I have addressed the issues. Please go through them once. @OrangeChannel @FichteFoll

File: src/typesetting/maskingProcedure.md
Comment (FichteFoll) [reply src/typesetting/maskingProcedure.md]: Fine with me, I'll just do a quick pass on some of the sentences myself.

**Review by FichteFoll — APPROVED** [review]
I did a quick pass on some of the phrasings, the folder structure, line breaks, etc. I didn't change anything about the contents, but please do a quick pass if there is anything amiss or that you don't agree with @gsk143 @OrangeChannel.

Otherwise I will merge this probably in a week or so.

Edit: Oh, I also ran the screenshots through `optipng`, significantly reducing their sizes (`pingo` doesn't build on the AUR currently).

**Comment by FichteFoll** [comment]
I also considered using git-lfs for the images, but it would make more sense if we had that for all images and I'd have to rewrite the git history for that anyway, so I can do that whenever I feel like it. It's not like the actual commits are important here and absolutely mustn't be rewritten.

**Pull request merged**
